docs/usage.md

```diff
@@ -102,6 +102,9 @@ Run casr-java:
 
     $ casr-java -o java.casrep -- java casr/tests/casr_tests/java/Test1.java
 
+You can specify paths to directories with Java source files via environment variable
+CASR\_SOURCE\_DIRS  (list separated by ':').
```

Comment:
We should set this variable in oss-sydr-fuzz/json-sanitizer Docker

Reply:
Maybe, we can also use this variable for source files detection in sydr-fuzz cov-html?